plugins/scaffolder/src/components/OngoingTask/OngoingTask.tsx

```diff
@@ -123,6 +126,15 @@ export const OngoingTask = (props: {
     }
   }, [taskStream.error, taskStream.completed]);
 
+  useEffect(() => {
+    const templateEntityRef = taskStream.task?.spec.templateInfo?.entityRef;
+    if (!templateEntityRef) {
+      return;
+    }
+    const presentation = entityPresentationApi.forEntity(templateEntityRef);
```

Comment:
I wonder if we actually want to use the `promise` here instead and use `useAsync` seen as the call to the catalog will be async, and if it's not in the cache then it's never going to reload this when the snapshot changes right?

It's just a side affect that this entity ref is called already and loaded into the `entityPresentationApiRef` cache?

@freben am I understanding this right?

Reply:
Yes it make more sense to use `useAsync` here, changed it!